src/modules/navigator/geofence_params.c

```diff
@@ -128,7 +128,9 @@ PARAM_DEFINE_FLOAT(GF_MAX_HOR_DIST, 0);
 PARAM_DEFINE_FLOAT(GF_MAX_VER_DIST, 0);
 
 /**
- * Use Pre-emptive geofence triggering
+ * [EXPERIMENTAL] Use Pre-emptive geofence triggering
+ *
+ * NOTE: This is an experimental feature and is known for causing flyaways on vehicles. Use at your own risk.
```

Comment:
Certainly better than nothing. It is unfortunate that the parameter tables are rendered as HTML tables so we can't use markdown formatting.

Perhaps something for 1.15 (as per [the discussion on feature support here](https://discuss.px4.io/t/px4-maintainers-call-may-30-2023/32372#fire-v115-next-release-improvement-discussion-8)) but perhaps we could add an `@experimental` tag in the parameter names to make it clear that the feature is experimental? (probably not, since a whole feature is experimental, not a particular parameter).

```suggestion
 * [EXPERIMENTAL] Use Pre-emptive geofence triggering
 *
 * WARNING: This experimental feature may cause flyaways. Use at your own risk.
```

Reply:
> Perhaps something for 1.15 (as per [the discussion on feature support here](https://discuss.px4.io/t/px4-maintainers-call-may-30-2023/32372#fire-v115-next-release-improvement-discussion-8)) but perhaps we could add an `@experimental` tag in the parameter names to make it clear that the feature is experimental?

I think that could be nice, if we have mechanism in QGC to prompt "WARNING! This feature is experimental!" to make sure user gets notified the fact that it's experimental properly.

Otherwise, I think you are probably thinking of documentation aspect of it as well, for displaying parameters with those tags with special colors perhaps?

Btw, displaying markdown formatting in HTML is possible, for example like this: https://www.makeuseof.com/md-block-render-markdown-web-page/